FileName.c

```diff
@@ -0,0 +1,42 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+
+#define MAX_CHARS 256
+
+
+char findMostFrequentChar(char* str, int len) 
+{
+    int i, j;
+    int max_count = 0; 

```

Comment:
```suggestion
    int maxCount = 0; 
```
Имена переменных должны быть в CamelCase, первая буква строчная.